incubator/kylo/values.yaml

```diff
@@ -0,0 +1,418 @@
+    pullPolicy: Always
+    tag: "v0.9.1-elasticsearch"
+
+  nodeSelector: {}
+
+  ## Configure Persistence Volumes / Claims
+  persistence:
+    enabled: true
+    ## A manually managed Persistent Volume and Claim
+    ## Requires persistence.enabled: true
+    ## If defined, PVC must be created manually before volume will be bound
+    # existingClaim:
+
+    ## NiFi data Persistent Volume Storage Class
+    ## If defined, storageClassName: <storageClass>
+    ## If set to "-", storageClassName: "", which disables dynamic provisioning
+    ## If undefined (the default) or set to null, no storageClassName spec is
+    ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
+    ##   GKE, AWS & OpenStack)
+    ##
+    # storageClass: "-"
+    annotations: {}
+    accessMode: ReadWriteOnce
+    size: 8Gi
+    volumes:
+    #  - name: nothing
+    #    emptyDir: {}
+    mounts:
+    #  - mountPath: /var/nothing
+    #    name: nothing
+    #    readOnly: true
+
+  ## Configure resource requests and limits
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  resources: {}
+
+## Kylo configuration
+kylo:
+  services:
+    image:
+      repository: govcloud/docker-kylo
+      pullPolicy: Always
+      tag: "v0.9.1-services"
+
+    nodeSelector: {}
+
+  ui:
+    image:
+      repository: govcloud/docker-kylo
+      pullPolicy: Always
+      tag: "v0.9.1-ui"
+
+    nodeSelector: {}
+
+  activeDirectory:
+    enabled: false
+
+  ## Configure Ingress resource that allow you to access the Kylo installation
+  ## Set up the URL
+  ## ref: http://kubernetes.io/docs/user-guide/ingress/
+  ##
+  ingress:
+    enabled: true
+    domain: kylo.govcloud.ca
+    annotations: {}
+      # kubernetes.io/ingress.class: nginx
+      # kubernetes.io/tls-acme: "true"
+    # hosts:
+    #  - chart-example.local
+    tls: []
+    #  - secretName: chart-example-tls
+    #    hosts:
+    #      - chart-example.local
+
+  ## Configure Persistence Volumes / Claims
+  persistence:
+    enabled: true
+    ## A manually managed Persistent Volume and Claim
+    ## Requires persistence.enabled: true
+    ## If defined, PVC must be created manually before volume will be bound
+    # existingClaim:
+
+    ## NiFi data Persistent Volume Storage Class
+    ## If defined, storageClassName: <storageClass>
+    ## If set to "-", storageClassName: "", which disables dynamic provisioning
+    ## If undefined (the default) or set to null, no storageClassName spec is
+    ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
+    ##   GKE, AWS & OpenStack)
+    ##
+    # storageClass: "-"
+    annotations: {}
+    accessMode: ReadWriteOnce
+    size: 8Gi
+    volumes:
+    #  - name: nothing
+    #    emptyDir: {}
+    mounts:
+    #  - mountPath: /var/nothing
+    #    name: nothing
+    #    readOnly: true
+
+  ## Configure resource requests and limits
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  resources:
```

Comment:
Memory and CPU limits we usually want to be commented out as you can never tell, by default, where this will be run. Minikube is different than a real cluster.

Is there a good reason these are here?